contrib/flo-scio_2.11/pom.xml

```diff
@@ -0,0 +1,108 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.spotify</groupId>
+    <artifactId>flo</artifactId>
+    <version>0.2.14-SNAPSHOT</version>
+    <relativePath>../..</relativePath>
+  </parent>
+
+  <name>Flo Scio</name>
+  <artifactId>flo-scio_2.11</artifactId>
+  <description>
+    Flo Scio support
+  </description>
+
+  <properties>
+    <scala.baseVersion>2.11</scala.baseVersion>
+    <scala.version>2.11.12</scala.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.spotify</groupId>
+      <artifactId>flo-scala_2.11</artifactId>
+      <version>0.2.14-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>com.spotify</groupId>
+      <artifactId>scio-core_2.11</artifactId>
+      <version>0.5.6</version>
```

Comment:
I think this might be hard to keep up to date, as scio/Beam sometimes introduce breaking changes. The risk I see is that whenever there is new beam/scio versions, users of `flo-scio` will have to wait for updates in this lib here.

Have we considered defining `flo-scio` into the internal scio mono repo or maybe as a project here: https://github.com/spotify/scio ?

Reply:
We're touching a small enough part of the scio/beam api that I'm hopeful breaking changes will be rare.

Moving `flo-scio` into the internal scio mono repo seems like a missed opportunity to have this be open source although the thought has occurred to me as well.

I guess an argument could be made for moving it to the oss `spotify/scio` repo but as flo is still in alpha with much of the api and internals in flux, we'd have the same problem of it being hard to have `spotify/scio` keep up. Also, I fear moving to `spotify/scio` now would slow down the development of `flo-scio`.

Reply:
Makes sense. I guess that is something to consider for the future.